app/cafe.py

```diff
@@ -0,0 +1,20 @@
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if not visitor.get("vaccine"):
+            raise NotVaccinatedError("All friends should be vaccinated.")
+        if visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError("The vaccine must not be expired!")
```

Comment:
you can init these messages in VactineError class with __str__ method like "All friends should be vaccinated"
![image](https://github.com/mate-academy/py-lockdown-cafe/assets/141478152/f0aef867-6b42-47cf-8bd2-a6300eb54418)
or inside child classes your massages, but it **up to you**